clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -5171,27 +5222,27 @@ private <K, V> void buildFetcher(MetricConfig metricConfig,
                                      SubscriptionState subscriptionState,
                                      LogContext logContext) {
         buildDependencies(metricConfig, metadataExpireMs, subscriptionState, logContext);
-        fetcher = new Fetcher<>(
-                new LogContext(),
-                consumerClient,
-                minBytes,
-                maxBytes,
-                maxWaitMs,
-                fetchSize,
-                maxPollRecords,
-                true, // check crc
-                "",
-                keyDeserializer,
-                valueDeserializer,
-                metadata,
-                subscriptions,
-                metrics,
-                metricsRegistry,
-                time,
-                retryBackoffMs,
-                requestTimeoutMs,
-                isolationLevel,
-                apiVersions);
+        fetcher = spy(new Fetcher<>(
+                        new LogContext(),
+                        consumerClient,
+                        minBytes,
```

Comment:
If we are doing this, could we adopt the new style directly:

```
new Fetcher(
    a,
    b,
    ....
);

Reply:
Sorry, I did not understand this comment. Was your suggestion about the indentation of the next line? I have reverted it to original i.e. 8 space indent.